guidelines/project_guidelines.md

```diff
@@ -15,7 +15,7 @@ We use a [minimal template](#minimal-contributing-guidelines) that points to our
 A **License**
 [GPLv3](http://gplv3.fsf.org/) is our _preferred_ license for code and [Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/) for creative works and documentation.
 
-  We use a [License and Copyright block](#license-and-copyright-readme-block) added to our READMEs, add a [LICENSE](https://github.com/edgi-govdata-archiving/overview/blob/master/LICENSE) to the repo, and occasionally include a [Code Header License block](#code-header-license-block).
```

Comment:
Not sure I understand -- you mean, in case text from `project_guidelines.md` gets copied around? Is that the pattern we are advocating: copying `project_guidelines.md` into other places? Sorry to be dense, but I feel like I'm maybe missing something